llms/README.md

```diff
@@ -101,10 +101,14 @@ Here are a few examples of Hugging Face models that work with this example:
 - [deepseek-ai/deepseek-coder-6.7b-instruct](https://huggingface.co/deepseek-ai/deepseek-coder-6.7b-instruct)
 - [01-ai/Yi-6B-Chat](https://huggingface.co/01-ai/Yi-6B-Chat)
 - [microsoft/phi-2](https://huggingface.co/microsoft/phi-2)
+- [Qwen/Qwen-7B](https://huggingface.co/Qwen/Qwen-7B)
 
 Most
 [Mistral](https://huggingface.co/models?library=transformers,safetensors&other=mistral&sort=trending),
 [Llama](https://huggingface.co/models?library=transformers,safetensors&other=llama&sort=trending),
-and
 [Phi-2](https://huggingface.co/models?library=transformers,safetensors&other=phi&sort=trending)
 style models should work out of the box.
+For [Qwen](https://huggingface.co/models?library=transformers,safetensors&other=qwen&sort=trending) style models, it is required to enable the `trust_remote_code` option and specify the `eos_token`. This ensures proper functioning of the tokenizer with Qwen models. You can do this by passing `--trust-remote-code` and `--eos-token "<eos_token>"` in the command line, or by setting these options in the Python API:
+```python
+model, tokenizer = load("qwen/Qwen-7B", tokenizer_config={"eos_token": "<eos_token>", "trust_remote_code": True})
```

Comment:
'eos_token' needs to be `<|endoftext|>` rather than `<eos_token>`

Reply:
Nice catch, will update it.